util/pastPerformance/testReleasesPerformance

```diff
@@ -198,6 +198,10 @@ testReleasePerformance
 export CHPL_TEST_PERF_DATE="12/14/23"
 export CHPL_HOME=$chpl_release_home/chapel-1.33.0
 testReleasePerformance
+
+export CHPL_TEST_PERF_DATE="3/21/24"
```

Comment:
I don't know that this matters, but it might be best to use "03" here to match the previous entries' format.

Reply:
I don't think it does because there were several older ones using this format, but I updated this and all the older ones too